### PR TITLE
utils: rename cloud-utils-*-SHADED.jar to cloud-utils-*-bundled.jar to fix the build error on Mac

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes the build error on MacOS

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.0.0:shade (rebuild-war) on project cloud-utils: Error creating shaded jar: Could not replace original artifact with shaded artifact!: /Users/nicolas/cloudstack/utils/target/cloud-utils-4.19.0.0-SNAPSHOT-shaded.jar (No such file or directory) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :cloud-utils
```

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan package

File: debian/rules [hunk 1]
```diff
@@ -135,7 +135,7 @@ override_dh_auto_install:
 	install -D systemvm/dist/* $(DESTDIR)/usr/share/$(PACKAGE)-common/vms/
 	# We need jasypt for cloud-install-sys-tmplt, so this is a nasty hack to get it into the right place
 	install -D agent/target/dependencies/jasypt-1.9.3.jar $(DESTDIR)/usr/share/$(PACKAGE)-common/lib
-	install -D utils/target/cloud-utils-$(VERSION)-SHADED.jar $(DESTDIR)/usr/share/$(PACKAGE)-common/lib/$(PACKAGE)-utils.jar
+	install -D utils/target/cloud-utils-$(VERSION)-SHADED-2.jar $(DESTDIR)/usr/share/$(PACKAGE)-common/lib/$(PACKAGE)-utils.jar
```
Comment (rohityadavcloud) [inline debian/rules:138]: any other way, perhaps we should call it `shaded` in filename?

File: utils/pom.xml [hunk 2]
```diff
@@ -261,7 +261,7 @@
                             <goal>shade</goal>
                         </goals>
                         <configuration>
-                            <finalName>${project.artifactId}-${project.version}-SHADED</finalName>
+                            <finalName>${project.artifactId}-${project.version}-SHADED-2</finalName>
```
Comment (rohityadavcloud) [inline utils/pom.xml:264]: can you advise what's causing the failures @weizhouapache - should we call it something else like `bundled` or `shaded` lowercase?

Comment (weizhouapache) [reply utils/pom.xml:264]: @rohityadavcloud 
maven will create `-shaded.jar` and copy to `-SHAED.jar`. it works on Linux.
I have no idea why it fails on MacOS.

`bundled` looks good. I will rename it.

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan LLpackage

**Comment by weizhouapache** [comment]
@blueorangutan LLtest matrix

**Comment by rohityadavcloud** [comment]
cc @nvazquez can you review/build locally on your mac, see if that works?

**Review by nvazquez — APPROVED** [review]
LGTM - tested OK on macos, can build properly - thanks for the fix @weizhouapache

**Review by harikrishna-patnala — APPROVED** [review]
LGTM.

verified the fix locally on my mac, build passed.

**Pull request merged**